src/app/park-form/park-form.component.html

```diff
@@ -1,111 +1,149 @@
+                        <label for="description">Description <span class="text-danger">*</span></label>
+                    </div>
+                    <div class='col-1 text-right mr-5'>
+                        <label
+                            [ngClass]="{'text-danger' : parkForm.get('description').value?.length > descriptionCharacterLimit}"
+                            for="description">{{ parkForm.get('description').value?.length }}/{{
+                            descriptionCharacterLimit }}</label>
                     </div>
-                    <textarea rows="5" type="text" class='form-control' id="description" formControlName="description"
-                        placeholder="Enter park description ({{ descriptionCharacterLimit }} character limit)"></textarea>
                 </div>
+                <textarea rows="5"
+                    type="text"
+                    class='form-control'
+                    id="description"
+                    formControlName="description"
+                    placeholder="Enter park description ({{ descriptionCharacterLimit }} character limit)"></textarea>
+            </div>
+        </form>
 
-            </form>
-            <div *ngIf="parkForm.invalid" class="row">
-                <div class='flex-fill'></div>
+        <div *ngIf="parkForm.invalid"
+            class="row">
+        </div>
+        <div class="row">
+            <div *ngIf="!isNewPark"
+                class="col-md-2">
+                <button class="btn btn-danger icon-button"
+                    (click)="delete()"><span class="material-icons mr-1">delete</span>Delete</button>
             </div>
-            <div class="row">
-                <div *ngIf="!isNewPark" class="col-md-2">
-                    <button class="btn btn-danger" (click)="delete()"><i
-                            class="material-icons mr-1">delete</i>Delete</button>
-                </div>
-                <div class='flex-fill'></div>
-                <div class="d-flex justify-content-md-end mx-3">
-                    <button class='btn btn-danger m-1' (click)="cancel()">Cancel</button>
-                    <button class='btn btn-secondary m-1' [disabled]="parkForm.pristine" (click)="resetForm()">Reset
+            <div class="col-md-10 mt-3 mt-md-0">
+                <div class="d-flex justify-content-md-end">
+                    <button class='btn btn-danger mr-1'
+                        (click)="cancel()">Cancel</button>
+                    <button class='btn btn-secondary mx-1'
+                        [disabled]="parkForm.pristine"
+                        (click)="resetForm()">Reset
                         Form</button>
-                    <button class='btn btn-primary m-1' [disabled]="parkForm.invalid" (click)="submitForm()">Save
+                    <button class='btn btn-primary ml-1'
+                        [disabled]="parkForm.invalid"
+                        (click)="submitForm()">Save
```

Comment:
this row of buttons is not properly right justified, but only on park-add (park-edit is fine) - i think removing the delete button in add is causing this

Reply:
068e8a9f780f2d0e3f81771cb8ee53f01755fa68